packages/fxa-auth-server/lib/routes/account.ts

```diff
@@ -649,6 +649,34 @@ export class AccountHandler {
       throw err;
     }
   }
+  
+  async setPasswordThirdParty(request: AuthRequest) {
```

Comment:
@vbudhram Just confirming, the difference between this and the already-available `createPassword` is that this one is unauthenticated? Maybe we should call it something similar. It looked like from Figma this was happening still on Backbone, so I am not sure I have a clear picture of the flow 🤔

Reply:
Wow, I completely missed that this route even existed 🥲. I should be using the `createPassword` route instead of this, I'll update.

Reply:
To aid in the confusion, we have `/account/set_password` which is used by subscription services to set a password with an OAuth token